main/res/values/strings.xml

```diff
@@ -791,6 +791,8 @@
     <string name="init_summary_bigSmileysOnMap">If enabled big emoticons will be shown for logged caches on the map instead of the cache type icon (requires restart).</string>
     <string name="init_global_wp_extraction_disable">Disable waypoint extraction</string>
     <string name="init_summary_global_wp_extraction_disable">Globally disable waypoint extraction from personal cache notes. It can still be enabled for individual caches by deactivating the checkbox below the personal cache note</string>
+    <string name="init_prevent_cache_note_merge">Prevent Merging of Personal Cache Notes</string>
```

Comment:
So much uppercase words -> "Prevent merging of personal cache notes" fits better with the other texts.

Reply:
I have absolutely no problem to rephrase those words to anything that makes it more readable and easier to understand though.

Reason why so much was uppercase is that in English speaking countries, words in titles are uppercase while they are lowercase in regular text.

Reply:
will commit your suggestion: Prevent merging of personal cache notes" fits better with the other texts

Reply:
Actually this topic made me thinking. Look at the preference screen:

![Screenshot from 2021-09-14 07-35-37](https://user-images.githubusercontent.com/5296333/133201097-dab46045-bdca-49bb-af7f-e82b7414081d.png)

First of all, you can see that I have made my new entry consistent with the one directly above (as suggested by @bekuno  in the review) with respect to capitalization of the first letter of the words, however, the rules in English writing are different: https://www.webucator.com/article/how-to-capitalize-headings-and-titles/

Consistent with that style we have: 

- Show Friends/Own <-- all first letters uppercase
- Last Details Page <- all first letters uppercase
- Show Direction
- Use Chrome Webview

but:

- Disable waypoint extraction <-- which is not consistent with what the referenced document says
- Prevent merging of personal cache notes <-- does not comply with the rules for headings and titles

I would say, we have to establish some rules and then make it all consistent throughout all views we render in cgeo. If those rule are in sync  with the referenced link above, then those two lines after "but" should be changed to:

 - Disable Waypoint Extraction
 - Prevent Merging of Personal Cache Notes